soroban-sdk/src/cost_estimate.rs

```diff
@@ -0,0 +1,111 @@
+use soroban_env_host::{fees::FeeConfiguration, FeeEstimate, InvocationResources};
```

Comment:
Can we move this file into the testutils directory? We keep as much as possible in there for testutils only stuff, and everything in that directory will be marked as testutils only in docs so folks don't think it's usable outside tests.

Reply:
Good point, done.